test/python/onnx_importer/import_smoke_test.py

```diff
@@ -44,155 +44,10 @@
 OUTPUT_PATH.mkdir(parents=True, exist_ok=True)
 
 TEST_CAST_XFAILS = [
-    "light_light_bvlc_alexnet",
-    "light_light_inception_v1",
-    "light_light_squeezenet",
-    "light_light_vgg19",
-    "node_test_affine_grid_2d_align_corners_expanded_model",
-    "node_test_affine_grid_2d_expanded_model",
-    "node_test_affine_grid_3d_align_corners_expanded_model",
-    "node_test_affine_grid_3d_expanded_model",
```

Comment:
Nice! These should also start passing in the downstream test suite I've been setting up for IREE to use: https://github.com/nod-ai/SHARK-TestSuite/blob/main/iree_tests/onnx/node/import_failures.txt

Reply:
Yup, should be expected. Hopefully this should be the last major changes there.